.gitignore

```diff
@@ -4,10 +4,10 @@
 **/pubspec.lock
 misc/
 .idea/
-**/*.g.dart
 download/
 lib/*.dll
 lib/*.dylib
 lib/*.so
 lib/*.a
 .vscode/
+**/*.g.dart
```

Comment:
Not sure about ignoring the generated files since they're not auto-refreshed on during build, but since it already was ignored, let's keep it for now

Reply:
Right, I had committed the `.g.dart` files into the repository before, but when I tried to rebuild them, `build_runner´ warned me that this file would be overwritten. This warning only occurred, when the file was version-controlled, so because it is essentially generated from the JSON file anyway, there's no need to have this redundant information in the repository.